google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITTransactionManagerTest.java

```diff
@@ -212,4 +214,31 @@ public void abortAndRetry() throws InterruptedException {
       manager2.close();
     }
   }
+
+  @SuppressWarnings("resource")
+  @Test
+  public void testTransactionManagerReturnsCommitStats() throws InterruptedException {
+    assumeFalse("Emulator does not return commit statistics", isUsingEmulator());
+    try (TransactionManager manager = client.transactionManager(Options.commitStats())) {
+      TransactionContext transaction = manager.begin();
+      while (true) {
+        transaction.buffer(
+            Mutation.newInsertBuilder("T")
+                .set("K")
+                .to("KeyCommitStats")
+                .set("BoolValue")
+                .to(true)
+                .build());
+        try {
+          manager.commit();
+          assertNotNull(manager.getCommitResponse().getCommitStats());
+          assertEquals(2L, manager.getCommitResponse().getCommitStats().getMutationCount());
+          break;
+        } catch (AbortedException e) {
+          Thread.sleep(e.getRetryDelayInMillis() / 1000);
```

Comment:
BUG! Thread.sleep takes milliseconds. No need to divide by 1000.

Reply:
Good point. Changed and [filed a bug](https://github.com/googleapis/java-spanner/issues/874) for the documentation that also includes this.